crates/editor/src/editor.rs

```diff
@@ -11050,6 +11050,129 @@ impl Editor {
         self.manipulate_immutable_lines(window, cx, |lines| lines.shuffle(&mut rand::rng()))
     }
 
+    pub fn rotate_forward(
```

Comment:
Same for these function names:

`rotate_selections_forward` / `rotate_selections_backward`